<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetDatabasePath</class>
 <widget class="QDialog" name="GetDatabasePath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Configuration</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>471</width>
     <height>191</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>461</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="ServerInfo">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="password">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="username">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="host">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="port">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>3306</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>password</zorder>
     <zorder>host</zorder>
     <zorder>port</zorder>
     <zorder>username</zorder>
    </widget>
    <widget class="QLabel" name="label_info">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter the configuration information of your SQL Server :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection to server successful, please select a Database :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="database_available">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Available Database :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>441</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The connection to database is successful. If database doesn't contains table RECIPE and table YEAST_TYPE, these two tables will be created. Here the actual table on your database :&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="table_available">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>431</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="next">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
